doc/warning.txt

```diff
@@ -0,0 +1,3 @@
+This example uploads data. For that reason, this example connects to the test server at test.openml.org.
```

Comment:
This is a bit nitpicky, but could you give the file a more descriptive name (e.g. `explain_use_test_server`) so that it's clear from the tutorial python files what the warning is generally about?

Reply:
Nono, you're right!
I too had the same thought so glad you're mentioning it :p